NEWS.md

```diff
@@ -16,7 +16,10 @@
 * Added `use_crate()` function to make adding dependencies to Cargo.toml easier within R, similar to `usethis::use_package()` (#361)
 * Fixed an issue in `rust_source()` family of functions that prevented usage of `r#` escape sequences in Rust function names (#374)
 * `use_cran_defaults()` now checks the `SystemRequirements` field in the `DESCRIPTION` file for cargo and rustc. It will display installation instructions if either is missing or provide the minimum required version if the installed version is outdated.
-* Added `use_msrv()` to aid in specifying the minimum supported rust version (MSRV) for an R package
+* Added `use_msrv()` to aid in specifying the minimum supported rust version
+  (MSRV) for an R package
```

Comment:
```suggestion
* Added `use_msrv()` to aid in specifying the minimum supported rust version (MSRV) for an R package
```

Reply:
I think there i an unintentional line break here.

Reply:
yeah, i saw that. have no idea how that happened.